tst/testinstall/grppc.tst

```diff
@@ -1,16 +1,70 @@
 #@local F,G,S,c,cl,f,g,g1,g10,g3,gens,h,hh,i,m,n,pcgs,r,rws,sys,u,v,x,y
 gap> START_TEST("grppc.tst");
+gap> Display(TrivialGroup());
+Trivial Group
 gap> h:=Group((1,2,3,4),(1,2));;
 gap> m:=IsomorphismPcGroup(h);;
 gap> hh:=Image(m,h);;
+gap> Display(hh);
+pc-group with relations:
+g1^2 = id
+g2^3 = id
+g3^2 = id
+g4^2 = id
+g2 ^ g1 = g2^2
```

Comment:
Why `g2 ^ g1` and not `g2^g1`?  Or conversely: Why `g1^2` and not `g1 ^ 2`?

Reply:
It's "g2^2" on the right side because gap prints terms of generators this way. (So this is not easy to change.)
The output of the left is constructed manually - the way @fingolfin wanted to have it, see #3538.

Reply:
I'm just going to skip the spaces on the left side of the equations.